#!/usr/bin/env python
PACKAGE = "gcop_ros_est"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#configure
gen.add("dyn_debug_on",    bool_t, 0, "Display debug data", True)

# configure ekf
gen.add("save_one_acc_reading",             bool_t, 0, "Save one accelerometer reading", False)
gen.add("dyn_enable_gps",             bool_t, 0, "Use gps in estimation or not. Not using gps will give orientation only", True)


# save calibration bool
#calibration file name string



exit(gen.generate(PACKAGE, "calibrator", "Calibrator"))
